Various/BuyOne_Convert custom actions to Lua ReaScript.lua

```diff
@@ -0,0 +1,14 @@
+-- @description Convert custom actions to Lua ReaScript
+-- @author BuyOne
+-- @version 1.2
+-- @metapackage
+-- @provides
+--   [main] . > BuyOne_Convert custom action to Lua ReaScript.lua
+--   [main] . > BuyOne_Convert custom actions in batch to Lua ReaScripts or dump as Lua code.lua
+--   [main] . > BuyOne_Convert filtered custom actions to Lua ReaScripts or dump as Lua code.lua
```

Comment:
This installs the Package file three times under different names. This technique is used when more than one script shares the same code. A script can detect its own filename at runtime using '({reaper.get_action_context()})[2]:match("([^/\\_]+)%.lua$")' and adapt its behavior. My "Move left/right edge of selected automation items to start/end of time selection" package uses this, for example.

However, because there is no code below, the three script files wouldn't do anything once installed.

Reply:
Got it.